### PR TITLE
Fix autocomplete mouse selection

### PR DESCRIPTION
**Pull request by vitormargis**
### What does this PR do:

Fixes an issue on automplete being unable to select options with mouse click
http://g.recordit.co/ExdlVG72Yx.gif

### Where should the reviewer start:

Check on acceptance if you're able to select from autocomplete dropdown with mouse click

### CODE REVIEW

**Comment by travix-frontend-services** [comment]
Deploy preview for *travix-ui-kit* ready!

Built with commit 604cf0739825472d0e3924d3d858261cdfd40cee

https://deploy-preview-51--travix-ui-kit.netlify.com

File: packages/travix-ui-kit/components/autoComplete/autoComplete.js [hunk 1]
```diff
@@ -181,6 +185,10 @@ class AutoComplete extends Component {
   applyActiveKey(e) {
     e.preventDefault();
 
+    if (!e.keyCode) {
+      return;
```
Comment (rbardini) [inline packages/travix-ui-kit/components/autoComplete/autoComplete.js]: This case is missing from unit tests, that's why CI is failing.

**Comment by rbardini** [comment]
@vitormargis when I click on a dropdown title, the dropdown doesn't close anymore, even if I click outside of it.

**Comment by joanrm20** [comment]
@rbardini @vitormargis shall we close this PR?

**Pull request closed without merging**